Directory.Build.props

```diff
@@ -29,15 +64,11 @@
       Packages which are intended to ship to NuGet.org must opt-in by setting this to true in their project file.
     -->
     <IsProductComponent>false</IsProductComponent>
```

Comment:
IsProductComponent is the KoreBuild version of IsShipping. Could we keep `IsProductComponent` as is and just put this code into Directory.Build.targets?
```xml
<IsShipping>$(IsProductComponent)</IsShipping>
```